src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const Key in robot) {
```

Comment:
```suggestion
  for (const key in robot) {
```
LowerCase for variable name